src/bridge/AbsOutbox.sol

```diff
@@ -76,6 +77,19 @@ abstract contract AbsOutbox is DelegateCallAware, IOutbox {
         rollup = address(_bridge.rollup());
     }
 
+    function postUpgradeInit() external onlyDelegated onlyProxyOwner {
```

Comment:
Since it only impacts existing ETH based rollups, we could put this implementation to `Outbox.sol` and revert if called on `ERC20Outbox.sol`?

Reply:
prefer to have it absoutbox, this is not specific to eth rollup, but rollup before 1.1.0